stock_no_negative/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Stock Disallow Negative
```

Comment:
@mreficent Hmm.. I think it is right. As this module is deported from v10 and over there it was like this.

Maybe @jbeficent knows better.